Dockerfile.base

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y -qq
+RUN apt-get install -y -qq \
+  build-essential \
+  curl \
+  libssl-dev \
+  libreadline-dev \
+  zlib1g-dev \
+  libxml2-dev \
+  libxslt-dev \
+  libpq-dev \
+  nodejs \
+  postgresql-client
+
+ENV RUBY_VERSION 2.1.10
+
+RUN curl -O https://cache.ruby-lang.org/pub/ruby/2.1/ruby-${RUBY_VERSION}.tar.bz2 &&\
```

Comment:
I like using rbenv (ruby-build) or rvm to install ruby. They contain knowledge of  where to get ruby source and how to compile. Maybe, next time we have issues of installing ruby.